packages/block-editor/src/components/border-radius-control/style.scss

```diff
@@ -11,9 +11,9 @@
 		align-items: flex-start;
 
 		> .components-unit-control-wrapper {
-			width: 110px;
+			width: 116px;
```

Comment:
I don't know if it's possible but ideally we shouldn't use absolute values like this. The controls should instead fill the width of the container, and the space between be determined via `gap`. This means that if we ever increase/decrease the width of the sidebar then everything will update gracefully.

Reply:
Thanks for flagging this 👍

We can avoid the hardcoded value for the `BorderRadiusControl` and so I've updated it in https://github.com/WordPress/gutenberg/pull/41860/commits/0f2b87c0a4af4f7cbfe67648a0eccc5567769db4. The `BorderBoxControl` is a little more complicated so I think that might be better addressed in a follow-up.